test/e2e/simpleflow_test.go

```diff
@@ -615,3 +615,14 @@ func TestNutanixKubernetes124SimpleFlowWithUUID(t *testing.T) {
 	)
 	runSimpleFlow(test)
 }
+
+func TestNutanixKubernetes125SimpleFlowWithUUID(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewNutanix(t, framework.WithUbuntu125NutanixUUID(),
+			framework.WithPrismElementClusterUUID(),
+			framework.WithSubnetUUID()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube125)),
```

Comment:
We have to add this line to set the feature flag for 1.25 on all of the 1.25 e2e tests in this PR. Else it'll fail at the validation level.

```suggestion
		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube125)),
framework.WithEnvVar(features.K8s125SupportEnvVar, "true"),
```